tests/decorators/test_short_circuit.py

```diff
@@ -30,49 +31,84 @@
 DEFAULT_DATE = datetime(2022, 8, 17)
 
 
-def test_short_circuit_decorator(dag_maker):
-    with dag_maker(serialized=True):
-
-        @task
-        def empty(): ...
+@pytest.mark.xfail(reason="This test might fail due to serialization issues")
+@pytest.mark.parametrize("condition, expected", [(True, State.SUCCESS), (False, State.SKIPPED)])
+def test_short_circuit_decorator(dag_maker, session, condition, expected):
+    with dag_maker(serialized=True, session=session):
 
         @task.short_circuit()
-        def short_circuit(condition):
+        def short_circuit():
```

Comment:
Why this change?

Reply:
`condition` is parametrized in the test, so I could pass it from the test scope, or using the overrides (original form).
Not sure why I changed it, so I restored the original form.